src/components/FormAlert/index.js

```diff
@@ -27,12 +21,11 @@ const Title = styled(Heading).attrs({
   forwardedAs: 'h4',
 })`
   font-family: Avenir Next LT W01 Demi, arial, sans-serif;
-  color: ${({ isNotification }) =>
-    isNotification ? themeColor('support', 'valid') : themeColor('secondary')};
+  color: ${({ isNotification }) => (isNotification ? themeColor('support', 'valid') : themeColor('secondary'))};
 
   ${({ hasMessage }) =>
     !hasMessage &&
-    css`
```

Comment:
Why is it necessary to remove the `css` function call? The [documentation still mentions it is required](https://styled-components.com/docs/api#css).

Reply:
It seems to fix the failing test.
The `css` template tag is not necessary, unless you interpolate a function.

> If you're interpolating a string you do not need to use this, only if you're
> interpolating a function.

I've added the change to #1002